src/hooks/useFetch.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+const useFetch = (url, lat, lng) => {
```

Comment:
Thank you so much for creating this custom hook! 😄

Reply:
@alxmcr , you asked for it 😄